### PR TITLE
Added dark theme

### PR DESCRIPTION
**Pull request by lucasamagno**
closed issue 2

### CODE REVIEW

File: SimpleValuation/wwwroot/css/site.css [hunk 1]
```diff
@@ -69,3 +78,7 @@ body {
   white-space: nowrap;
   line-height: 60px; /* Vertically center the text there */
 }
+
+input[type=range] {
```
Comment (DylanRichards) [inline SimpleValuation/wwwroot/css/site.css]: It doesn't work. Remove, please 😄

File: SimpleValuation/wwwroot/css/site.css [hunk 2]
```diff
@@ -69,3 +78,7 @@ body {
...
+    color: green;
+}
```
Comment (dylanrichards) [inline SimpleValuation/wwwroot/css/site.css]: It doesn't work. Remove, please 😄

**Review by dylanrichards — CHANGES_REQUESTED** [review]
Resolve comment about removing input color.

**Review by dylanrichards — APPROVED** [review]
Thank you! 👍

**Pull request merged**